src/Employer/Employer.Web/Mappings/VacancyDisplayViewModelMapper.cs

```diff
@@ -19,22 +19,22 @@ public class DisplayVacancyViewModelMapper
         private const int MapImageHeight = 256;
         private readonly IGeocodeImageService _mapService;
         private readonly IGetMinimumWages _wageService;
-        private readonly QualificationsConfiguration _qualificationsConfiguration;
         private readonly ExternalLinksConfiguration _externalLinksConfiguration;
         private readonly IEmployerVacancyClient _client;
+        private readonly IList<string> _qualifications;
 
         public DisplayVacancyViewModelMapper(
                 IGeocodeImageService mapService,
                 IGetMinimumWages wageService, 
-                IOptions<QualificationsConfiguration> qualificationsConfigOptions,
                 IOptions<ExternalLinksConfiguration> externalLinksOptions,
                 IEmployerVacancyClient client)
         {
             _mapService = mapService;
             _wageService = wageService;
-            _qualificationsConfiguration = qualificationsConfigOptions.Value;
             _externalLinksConfiguration = externalLinksOptions.Value;
             _client = client;
+
+            _qualifications = _client.GetCandidateQualificationsAsync().Result.QualificationTypes;
```

Comment:
Could we not use Lazy here and in the ctor for ReviewMapper

Reply:
Yes we could do. What I meant to do was move it into the mapping method like we do with the programmes. Or we could move both to be lazy initialised. I guess using the lazy initializer makes it easier to re-use if other methods are added to the class.